src/InvokePlaster.ps1

```diff
@@ -826,6 +827,21 @@ function Invoke-Plaster {
                 if (![string]::IsNullOrWhiteSpace($description)) {
                     $newModuleManifestParams['Description'] = $description
                 }
+                if (![string]::IsNullOrWhiteSpace($copyright)) {
+                    $newModuleManifestParams['Copyright'] = $copyright
+                }
+                if (![string]::IsNullOrWhiteSpace($projectUri)) {
+                    $newModuleManifestParams['projectUri'] = $projectUri
```

Comment:
Not that it matters (PS being case-insensitive), I would PascalCase `projectUri` (to `ProjectUri`) along with the rest of the hashtable keys below.